eve/main.yml

```diff
@@ -234,6 +234,37 @@ stages:
       - ShellCommand: *retrieve_iso
       - ShellCommand: *retrieve_iso_checksum
       - ShellCommand: *check_iso_checksum
+      - ShellCommand:
+          name: Check if unzip and curl is installed
+          command: |-
+            if ! unzip -h 2&> /dev/null; then
+              echo "aborting - unzip not installed and required" >&2
+              exit 1
+            fi
+            if ! curl -h 2&> /dev/null; then
+              echo "aborting - curl not installed and required" >&2
+              exit 1
+            fi
+          haltOnFailure: true
+      - ShellCommand:
+          name: Download and install terraform
+          command: >
+            TERRAFORM_VERSION=0.12.3
+            tf_url="https://releases.hashicorp.com/terraform/";
+            tf_suburl="${TERRAFORM_VERSION}/";
+            tf_file="terraform_${TERRAFORM_VERSION}_linux_amd64.zip";
```

Comment:
optional: If you want, you can use the `env` option on the `ShellCommand` step to set those variables.

Like the following:
```yaml
ShellCommand:
    name: doing stuff
    command: do stuff
    env:
      TERRAFORM_VERSION: '0.12.3'
...
```